addons/website/controllers/form.py

```diff
@@ -173,6 +173,10 @@ def extract_data(self, model, values):
                 except ValueError:
                     error_fields.append(field_name)
 
+                if dest_model._name == 'mail.mail' and field_name == 'email_from':
+                    # Always add the email in the custom message
+                    custom_fields.append((_('email'), field_value))
```

Comment:
Should we capitalize "Email" ?
Maybe not, not sure how it looks in the email body for other custom fields.